qiskit_experiments/data_processing/processor_library.py

```diff
@@ -49,8 +57,12 @@ def get_processor(
     meas_return = run_options.get("meas_return", MeasReturnType.AVERAGE)
     normalize = analysis_options.get("normalization", True)
 
+    num_qubits = experiment_data.metadata.get("num_qubits", 1)
+
+    outcome = analysis_options.get("outcome", "1" * num_qubits)
```

Comment:
`num_qubits` and `outcome` are only used when `meas_level == MeasLevel.CLASSIFIED` thus these variables are preferred to be defined in that context. Other changes looks good. You may also want to upgrade this in future (not necessary by this PR since we need some mechanism to pass target label if not "1").
https://github.com/Qiskit/qiskit-experiments/blob/c211b0175eb077edcb9d8810a9694272f81a8b44/qiskit_experiments/library/randomized_benchmarking/rb_experiment.py#L94-L99

Reply:
Thanks! Implemented in 9bd06fa.